src/dictionary/dictionary.h

```diff
@@ -374,10 +374,8 @@ dictionary_find(
 
 /**
 @brief		Tests the supplied @p key against the predicate registered in the
-			cursor for the given @p parent type.
+			cursor for equality or to see if it is in the specified upper and lower bounds for the supplied @p cursor.
```

Comment:
I would write this as, `Tests the predicate stored within the given @p cursor, using the given @p key as the argument.`, since the cursor could potentially be something different from an equality or a range cursor.